model/ftn/w3iogomd.ftn

```diff
@@ -2604,6 +2615,90 @@
 !
         END IF
 !
+!  IN CASE OF GENERATION OF A NEW FILE OUTPUT EVERY DELTA OUTPUT
+! open file ---------------------------------------------------------- *
+! ( IPASS = 1 )
+!
+      IF ( IPASS.GE.1 .AND. OFILES(1) .EQ. 1) THEN
+          WRITE  = INXOUT.EQ.'WRITE'
+        ELSE
+          IF ( WRITE .AND. INXOUT.EQ.'READ' ) THEN
+              WRITE (NDSE,901) INXOUT
+              CALL EXTCDE ( 2 )
+            END IF
+        END IF
+
+!
+      IF ( IPASS.GE.1 .AND. OFILES(1) .EQ. 1) THEN
+          I      = LEN_TRIM(FILEXT)
+          J      = LEN_TRIM(FNMPRE)
+!
+
+!!          write(*,*) '===FCSTTIME: ',FCSTTIME
+          IF (FCSTTIME .LT. 10.) THEN
+            CO='00'
+          ELSE IF (FCSTTIME .GE. 10. .AND. FCSTTIME .LT. 100.) THEN
+            CO='0'
+          ELSE IF (FCSTTIME .GE. 100) THEN
+            CO=''
+          END IF
+    
+! Convert integer FCSTTIME into a character Hstr with 3 columns 
+          WRITE(Hstr,"(f6.2)") FCSTTIME
+          Hstr(4:4)='p'
+          FCSTNAM='f'//TRIM(ADJUSTL(CO))//TRIM(ADJUSTL(Hstr))
+!!          write(*,*)'w3iogo OutPut Filename: out_grd.', FCSTNAM,'.',FILEXT(:I)
```

Comment:
Was there any consideration to putting the valid time instead of a forecast hour as suggested here: https://github.com/NOAA-EMC/WW3/issues/1